src/domUpdates.js

```diff
@@ -4,26 +4,26 @@ class DomUpdates {
   constructor() {
     this.name = 'Dom'
   }
-  
-  populateCards(recipes, cardArea, selectedIds, button, propertyName){
+
+  populateCards(recipes, cardArea, selectedIds, button, propertyName) {
     cardArea.innerHTML = '';
     recipes.forEach(recipe => {
       let buttonStatus, message, icon;
       if (selectedIds && selectedIds.includes(recipe.id) && propertyName === 'favoriteRecipes') {
         // if(propertyName === 'favoriteRecipes'){
-          buttonStatus = `favorite-active`
-          message = 'Favorited'
-          icon = `<i class="fas fa-utensil-spoon"></i>`
-        } else {
+        buttonStatus = `favorite-active`
+        message = 'Favorited'
+        icon = `<i class="fas fa-utensil-spoon icon" id = "${recipe.id}"></i>`
+      } else {
```

Comment:
Wonder if there is a better way we can access these ideas here, I remember certain dom properties that could search children or parent elements. Just a thought to avoid putting ID everywhere.

Reply:
I saw that last night when you were trying to access the information form the parent, I will look into it ina bit and we get back together we could probably refactor a little bit